### DISCUSSION TITLE
How to enable <Tab> in cmdline?

### DISCUSSION THREAD

**Discussion by kleinepls** [saghen/blink.cmp#807] [Q&A] [answered]
How can I have the default tab behavior in cmdline? I would frequently update my packages by typing `lazy` and then hitting tab to turn it to `Lazy`, and I noticed that this doesn't work anymore ever since I moved to blink. And of course any other suggestions.

**Comment by Quitlox** [comment, marked as answer]
You can change the behaviour for the cmdline mode using the `keymap.cmdline` option. It sounds like you're looking for `select_and_accept`.

```lua
require("blink-cmp").setup({
    keymap = {
        cmdline = {
            ["<Tab>"] = { "select_and_accept", "fallback" },
        },
    },
})
```

This can all be found in the documentation: https://cmp.saghen.dev/configuration/keymap.html
